mozci/console/commands/push.py

```diff
@@ -172,23 +172,17 @@ def handle(self) -> None:
             )
 
         for push in pushes:
-            try:
-                classification, regressions = push.classify(
-                    intermittent_confidence_threshold=medium_conf,
-                    real_confidence_threshold=high_conf,
-                )
-                if retrigger_unknown:
-                    for _, tasks in regressions.unknown.items():
-                        retrigger(tasks=tasks, repeat_retrigger=1)
-                self.line(
-                    f"Push associated with the head revision {push.rev} on "
-                    f"the branch {self.branch} is classified as {classification.name}"
-                )
-            except Exception as e:
```

Comment:
There are a few other places in this file where we are catching an exception and just logging it and returning. We should either remove the exception catching, or re-raise the exception after logging.

Reply:
Done and pushed, please tell me if I missed some again...